src/Microsoft.Health.Dicom.Core/Exceptions/Validation/ExceedMaxLengthException.cs

```diff
@@ -0,0 +1,28 @@
+﻿// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+using System.Globalization;
+using Dicom;
+using Microsoft.Health.Dicom.Core.Features.Validation;
+
+namespace Microsoft.Health.Dicom.Core.Exceptions.Validation
+{
+    public class ExceedMaxLengthException : ElementValidationException
+    {
+        public ExceedMaxLengthException(string name, DicomVR vr, string value, int maxlength) :
+            base(
```

Comment:
If you do decide to keep these exceptions, a couple formatting guidelines:
1. If the call to the base ctor is too long for the line, move `: base` to the next line.
2. Ensure each line is only idented by 4 spaces and do not attempt to align it vertically (most of the time)
3. Typically, the last `)` is kept on the same line.

Eg.
```csharp
public ExceedMaxLengthException(string name, DicomVR vr, string value, int maxlength)
    : base(
        name,
        vr,
        value,
        ValidationErrorCode.ExceedMaxLength,
        string.Format(CultureInfo.InvariantCulture, DicomCoreResource.ErrorMessageExceedMaxLength, maxlength))
{
    // TODO: ...
}
```
Hopefully we can also keep the call to the base ctor on one line at most lol